.github/workflows/libjs-test262.yml

```diff
@@ -64,29 +64,22 @@ jobs:
 
       - name: Create build directory
         run: |
-          mkdir -p Build/lagom
-          mkdir -p Build/lagom/UCD
+          mkdir -p Build/UCD
 
       - name: UnicodeData cache
         # TODO: Change the version to the released version when https://github.com/actions/cache/pull/489 (or 571) is merged.
         uses: actions/cache@03e00da99d75a2204924908e1cca7902cafce66b
         with:
-          path: ${{ github.workspace }}/Build/lagom/UCD
+          path: ${{ github.workspace }}/Build/UCD
           key: UnicodeData-${{ hashFiles('Userland/Libraries/LibUnicode/unicode_data.cmake') }}
 
-      - name: Build test-js
-        run: |
-          cd Build/lagom
-          cmake -GNinja -DCMAKE_C_COMPILER=gcc-10 -DCMAKE_CXX_COMPILER=g++-10 -DBUILD_LAGOM=ON ../../Meta/Lagom
-          ninja test-js
-
-      - name: Build libjs-test262-runner
+      - name: Build libjs-test262-runner and test-js
         working-directory: libjs-test262
         run: |
           mkdir -p Build
```

Comment:
No need for this one, due to the `Create build directory` step

Reply:
Oh, but there is! Because I messed up my rebase on your cache changes. The create build directory step up top should be based on libjs-test262/Build/UCD, not Build/UCD. Push incoming...